Limelight/Input/StreamView.m

```diff
@@ -49,11 +52,25 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
         UITouch *touch = [[event allTouches] anyObject];
         touchLocation = [touch locationInView:self];
         touchMoved = false;
+        dragTimer = [NSTimer scheduledTimerWithTimeInterval:2.0
+                                                     target:self
+                                                   selector:@selector(onDragStart:)
+                                                   userInfo:nil
+                                                    repeats:NO];
+    }
+}
+
+- (void)onDragStart:(NSTimer*)timer {
+    if (!touchMoved && !isDragging){
```

Comment:
I corrected this part to start the timer only if you touch with one finger.

But, do you have something for testing the app ? 
My problem is each time i want to test something, i have to reboot the computer under windows to launch geforce experience..